.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,63 @@
+name: Bump Patch Version
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - .*
+      - .cruft.json
+      - .github/**.md
+      - .github/**.yml
+      - AUTHORS.rst
+      - CHANGES.rst
+      - Dockerfile
+      - MANIFEST.in
+      - Makefile
+      - README.rst
+      - docker-compose.yml
+      - docs/**.ico
+      - docs/**.ipynb
+      - docs/**.py
+      - docs/**.rst
+      - docs/**.svg
+      - docs/**.txt
+      - docs/Makefile
+      - etc/**.cfg
+      - environment-docs.yml
+      - environment.yml
+      - finch/__version__.py
+      - setup.cfg
+      - setup.py
+
+jobs:
+  bump_patch_version:
```

Comment:
So I guess this bump patch version only, from the job name.  What if we want to bump the minor or major?  We have a way to tell the workflow to not perform automatically the patch bump so we can manually perform minor or major bump?

Reply:
My understanding would be that we would perform the changes for a major/minor release within a PR, bumping the version ourselves.

This PR will always touch 5 files:
  - .cruft.json
  - CHANGES.rst
  - Dockerfile
  - docs/source/conf.py
  - finch/__version__.py

Changes to those files will not trigger a bump version, but changes to all those files will trigger the tagging action.

Reply:
So if we do it manually, we will also change all those files, which would trigger the tagging action again?

Reply:
If we only trigger on push to "master", the action will not trigger on changes within a branch. I think the most important thing to take note of is that a "Prepare release" PR should not touch code.

_But what if it does?_

This is quickly becoming complicated. I may need to rethink this approach.